doc/drone-dev-env-setup-guide/drone-dev-env-setup-guide.md

```diff
@@ -209,16 +206,16 @@ dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /nores
 ```
 
 PC再起動後、PowerShellを開いて次のコマンドを実行しPowerShellを閉じてください。これでデフォルトのバージョンを1にします。  
-デフォルトが1で困る場合は、WSLバージョン2でインストールした後で個別に `wsl --set-version Ubuntu-20.04 1` を実行して変更してください。
+デフォルトが1で困る場合は、WSLバージョン2でインストールした後で個別に `wsl --set-version Ubuntu22.04 1` を実行して変更してください。
 ```powershell
-wsl --set-default-version 1
+wsl --set-default-version 2
 ```
-## 4.2. Ubuntu20のインストールと初期設定
-【注意】すでにUbuntu 20.04がインストール済みの場合はスキップしてください。
+## 4.2. Ubuntu22のインストールと初期設定
+【注意】すでにUbuntu22.04がインストール済みの場合はスキップしてください。
 
-PowerShellを開いて次のコマンドを実行して`Ubuntu 20.04`をインストールしてください。
+PowerShellを開いて次のコマンドを実行して`Ubuntu22.04`をインストールしてください。
 ```powershell
-wsl --install -d Ubuntu-20.04
+wsl --install -d Ubuntu22.04
 ```
 
 インストールが完了したら `PCを再起動` をしてください。<br/>

```

Comment:
インストールの完了と同時にUbuntuのウインドウが出現し以下のエラーが出力されました。
```
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x800701bc
Error: 0x800701bc WSL 2 ???????????? ??????????????????????? https://aka.ms/wsl2kernel ?????????

Press any key to continue...
```
ChatGPT先生に訊いてみたところ、Linux カーネルの更新が必要かもしれないとのことでしたので、
https://learn.microsoft.com/ja-jp/windows/wsl/install-manual#step-4---download-the-linux-kernel-update-package
ここの【手順 4 - Linux カーネル更新プログラム パッケージをダウンロードする】を実施した上で、あらためて
`wsl --install -d Ubuntu-22.04`
を実行したところ、`Installing, this may take a few minutes…`が表示されたまま10分ほど経っても次に進まなかったため、しびれを切らしPCを再起動しました。

再度、
`wsl --install -d Ubuntu-22.04`
を実行すると、無事Ubuntuが立ち上がりましたが、何故か username と password を訊かれることなく `root` ユーザーでUbuntuが起動しました。

この原因は調査中です・・・

Reply:
本エラー発生時の対応法を追記しました。